tests/translated_table_model_tester.ts

```diff
@@ -1,5 +1,235 @@
 import { Expect, Test } from 'alsatian';
+import { ArrayTableModel, AddRowOperation, Operation, RemoveRowOperation,
+  TranslatedTableModel, UpdateValueOperation } from '../source';
 
 /** Tests the TranslatedTableModel. */
 export class TranslatedTableModelTester {
+
+  /** Tests if the number of rows is correct */
+  @Test()
+  public testRowCount(): void {
+    const translatedTable = new TranslatedTableModel(new ArrayTableModel());
+    Expect(translatedTable.rowCount).toEqual(0);
+    const model = new ArrayTableModel();
+    model.addRow([1, 2 ,3]);
+    model.addRow([4, 5, 6]);
+    model.addRow([7, 8, 9]);
+    const translatedTable2 = new TranslatedTableModel(model);
+    Expect(translatedTable2.rowCount).toEqual(3);
+  }
+
+  /** Tests if the number of columns is corrrect. */
+  @Test()
+  public testColumnCount(): void {
+    const translatedTable = new TranslatedTableModel(new ArrayTableModel());
+    Expect(translatedTable.columnCount).toEqual(0);
+    const model = new ArrayTableModel();
+    model.addRow([1, 2, 3, 0]);
+    model.addRow([4, 5, 6, 0]);
+    model.addRow([7, 8, 9, 0]);
+    const translatedTable2 = new TranslatedTableModel(model);
+    Expect(translatedTable2.columnCount).toEqual(4);
+  }
+
+  /** Tests if the get function works. */
+  @Test()
+  public testGet(): void {
+    const model = new ArrayTableModel();
+    model.addRow([1, 2, 3, 4, 5]);
+    model.addRow([6, 7, 8, 9, 10]);
+    const translatedTable = new TranslatedTableModel(model);
+    Expect(translatedTable.get(0,1)).toEqual(2);
+    Expect(translatedTable.get(1,3)).toEqual(9);
+  }
+
+  /** Tests the move function. */
+  @Test()
+  public testMoveRow(): void {
+    const model = new ArrayTableModel();
+    model.addRow([1, 2, 1, 2]);
```

Comment:
There's no need to add multiple columns, it makes it harder to test as well. One single column is sufficient to ensure correctness.